lightspeed/assemblies/assembly_configuring-vscode-extension.adoc

```diff
@@ -13,7 +13,7 @@ ifdef::context[:parent-context: {context}]
 
 To access {LightspeedShortName}, all Ansible users must install and configure the Ansible VS Code extension in their VS Code. The Ansible VS Code extension uses the Ansible-specific IBM watsonx Granite model configured in the {LightspeedShortName} administrator portal as the default mode for all users in your organization. 
 
-You can also use a custom, fine-tuned model if your organization administrator has created a custom model and has shared the model ID with you separately. You use the *model-override* setting in the Ansible VS Code extension to override the default model, and use the custom model instead. Using a custom model enables you to improve the code recommendation experience and tune the model to your organizational automation patterns. For example, if you are using {LightspeedShortName} both as an organization administrator and an user, you can test the custom model for select Ansible users before making it available for all users in your organization. For more information, see xref:configuring-custom-models_lightspeed-user-guide[Configuring custom models]. 
+You can also use a custom, fine-tuned model if your organization administrator has created a custom model and has shared the model ID with you separately. You use the *model-override* setting in the Ansible VS Code extension to override the default model, and use the custom model instead. Using a custom model enables you to improve the code recommendation experience and tune the model to your organizational automation patterns. For example, if you are using {LightspeedShortName} both as an organization administrator and an user, you can test the custom model for select Ansible users before making it available for all users in your organization. For more information, see xref:configuring-custom-models_set-up-lightspeed[Configuring custom models]. 
```

Comment:
I would remove the "you" from the second sentence so it reads "Use the model override setting...etc"

Fourth sentence: Change "an" to "a." Should be "both as an organization administrator and **a** user"

Reply:
Done